src/documentation/chunk.rs

```diff
@@ -107,80 +107,85 @@ impl CheckableChunk {
             &range
         );
 
-        let Range { start, end } = range.clone();
-        let mut active = false;
+        let Range { start, end } = range;
         self.source_mapping
             .iter()
-            .skip_while(|(sub, _span)| sub.end <= start)
-            .take_while(|(sub, _span)| end <= sub.end)
+            .skip_while(|(fragment_range, _span)| fragment_range.end <= start)
+            .take_while(|(fragment_range, _span)| end <= fragment_range.end)
             .inspect(|x| {
                 trace!(">>> item {:?} ∈ {:?}", &range, x.0);
             })
-            .filter(|(sub, _)| {
+            .filter(|(fragment_range, _)| {
                 // could possibly happen on empty documentation lines with `///`
-                sub.len() > 0
+                fragment_range.len() > 0
             })
-            .filter_map(|(sub, span)| {
-                if span.start.line == span.end.line {
-                    debug_assert!(span.start.column <= span.end.column);
-                    if span.start.column > span.end.column {
-                        return None;
-                    }
+            .filter_map(|(fragment_range, fragment_span)| {
+                // trim range so we only capture the relevant part
```

Comment:
what is considered an irrelevant part?

Reply:
It's not about irrelevant, it's a coverage problem of the given `input` range on the set of fragmented ranges. Note the fragmentation is caused by clustering of multiple literals. A fragment is a mere description of a range/span couple concept of a chunk.